ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/cluster-density-v2/cloud-bulldozer-e2e-benchmarking-cluster-density-v2-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+if [[ "$JOB_TYPE" == "presubmit" ]] && [[ "$REPO_OWNER" = "cloud-bulldozer" ]] && [[ "$REPO_NAME" = "e2e-benchmarking" ]]; then
+    PR_LINK="https://github.com/${REPO_OWNER-}/${REPO_NAME-}/pull/${PULL_NUMBER-}"
```

Comment:
This git URL is not correct to fetch PRs

Reply:
One correct way to do this below:

```shell
$ git clone https://github.com/cloud-bulldozer/e2e-benchmarking 
$ git pull origin pull/${PULL_NUMBER}/head:${PULL_NUMBER}
$ git switch ${PULL_NUMBER}
```

Reply:
Proof
```shell
$ git pull origin pull/634/head:634 --rebase                            
remote: Enumerating objects: 13, done.             
remote: Counting objects: 100% (13/13), done.                                                                          
remote: Compressing objects: 100% (8/8), done.
remote: Total 13 (delta 5), reused 12 (delta 5), pack-reused 0 
Unpacking objects: 100% (13/13), 3.44 KiB | 3.44 MiB/s, done.
From https://github.com/cloud-bulldozer/e2e-benchmarking                                                               
 * [new ref]         refs/pull/634/head -> 634                                                                         
Successfully rebased and updated refs/heads/master.    
rsevilla@wonderland ~/e2e-benchmarking (master) $ git switch 634                                                               
Switched to branch '634'                        
rsevilla@wonderland ~/e2e-benchmarking (634) $ git log --oneline                                                        
b7dd7f6 (HEAD -> 634) Merge branch 'cloud-bulldozer:master' into ingress-perf-config-template
ae79591 (tag: v2.1.4) Deprecate legacy kube-burner directory (#608)
etc.
```

Reply:
Thanks for catching this. Tested and verified in local.